presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcWarnings.java

```diff
@@ -96,6 +98,16 @@ public void setup()
     {
         connection = createConnection();
         statement = connection.createStatement();
+
+        statement.execute("CREATE SCHEMA IF NOT EXISTS blackhole.blackhole");
+        statement.executeUpdate("" +
+                "CREATE TABLE IF NOT EXISTS blackhole.blackhole.slow_table (x int) " +
```

Comment:
There is no need to try-recreate this table per test method. Why not run this in `@BeforeClass`?